hack/run-functests.sh

```diff
@@ -29,4 +29,4 @@ echo "Running Functional Tests: ${GINKGO_SUITS}"
 # --failFast: ginkgo will stop the suite right after the first spec failure
 # --flakeAttempts: rerun the test if it fails
 # -requireSuite: fail if tests are not executed because of missing suite
-GOFLAGS=-mod=vendor ginkgo $NO_COLOR --v -r --failFast -skipPackage="5_latency_testing" --flakeAttempts=2 -requireSuite ${GINKGO_SUITS} -- -junitDir /tmp/artifacts
+GOFLAGS=-mod=vendor ginkgo $NO_COLOR --v -r --failFast --skipPackage="5_latency_testing,2_performance_update" --flakeAttempts=2 -requireSuite ${GINKGO_SUITS} -- -junitDir /tmp/artifacts
```

Comment:
Is there a consensus to always skip the update tests and run them separately in special use cases? cc @mrniranjan @MarSik

Reply:
I don't think you should add the extra `-` in `--skipPackage`

Reply:
I think the correct one is with two - https://onsi.github.io/ginkgo/